apps/main/src/llamalend/features/market-list/LlamaMarketsTable.tsx

```diff
@@ -71,7 +70,7 @@ export const LlamaMarketsTable = ({
   // Filter out markets after a certain creation date, because they're unsafe to use until Llamalend V2 is live.
```

Comment:
Now that `MARKET_CUTOFF_DATE` has been moved to be re-used in some other places too, it's probably better if we move the docs to the constant so that this information isn't lost when you encounter it being used in another place